### DISCUSSION TITLE
Fooocus is Still Updating...

### DISCUSSION THREAD

**Discussion by DavidDragonsage** [lllyasviel/Fooocus#3544] [General]
I noticed that when starting Fooocus this morning there was a "Fast-forward merge", as shown in the attachment.

There must be a ghost in the machine! 👻🙂

![GhostInTheMachine](https://github.com/user-attachments/assets/effe88cb-2783-40d1-a774-d7c1fe9d959c)

**Comment by Hypersonic-Popcorn** [comment]
According to the Github Blame feature, Illyasviel updated the ReadMe with a note the Fooous was in Limited LTS with bug fixes only. I assume this update is their work.

**Reply by DavidDragonsage** [reply]
@ForgetfulWasAria  Thank you, that was very helpful!

This link shows the ReadMe updates, with the changes in green. LTS means "limited long-term support":
https://github.com/metercai/SimpleSDXL/compare/SimpleSDXL...lllyasviel%3AFooocus%3Amain
